packages/applicant/src/pages/applications/index.page.tsx

```diff
@@ -21,15 +20,19 @@ export const getServerSideProps: GetServerSideProps<ApplicationsPage> = async ({
   };
 };
 
-const ExistingApplications = ({ applicationData }: ApplicationsPage) => {
+const ExistingApplications = ({
+  applicationData,
+  isUserLoggedIn,
+}: ApplicationsPage & {
+  isUserLoggedIn: boolean;
```

Comment:
Would declare this as a well named type that we can reuse. Something like `GetInitialPropsType`

Reply:
Good shout :)